api/presenters/PostPresenter.js

```diff
@@ -76,10 +85,10 @@ var postAttributes = (post, opts) => {
 // so that it knows when to stop expecting more.
 // we can't always use a naive approach to pagination, because
 // the order of results could shift while searching.
-var mapPresentWithTotal = function (posts, opts) {
+var mapPresentWithTotal = function (posts, userId, opts) {
```

Comment:
i would actually just drop this method and move the implementation into `PostController` -- since it's not used anywhere else, removing it would be one less layer of indirection